tests/functional/test_request_pages.py

```diff
@@ -671,3 +671,96 @@ def test_request_uploaded_files_status(
             expect(release_files_button).to_be_enabled()
         else:
             expect(release_files_button).not_to_be_enabled()
+
+
+def test_request_uploaded_files_counts(
+    live_server,
+    context,
+    page,
+    bll,
+    mock_old_api,
+    mock_notifications,
+):
+    # make a release request in APPROVED status
+    release_request = factories.create_request_at_status(
+        "workspace",
+        status=RequestStatus.APPROVED,
+        files=[
+            factories.request_file(
+                group="group", contents="1", path="file1.txt", approved=True
+            ),
+            factories.request_file(
+                group="group", contents="2", path="file2.txt", approved=True
+            ),
+            factories.request_file(
+                group="group", contents="3", path="file3.txt", approved=True
+            ),
+        ],
+    )
+    output_checker = login_as_user(
```

Comment:
Have you seen the `create_api_user()` factory function Simon recently added. Example in the tests here:
https://github.com/opensafely-core/airlock/blob/1c371d914380d68d01dfa2dc0bce3052bde69055/tests/functional/test_request_pages.py#L151-L154

Reply:
Oh yes, I have, I just haven't got used to using it yet, and i think I possibly wrote this test (i.e. copied/pasted the setup from another test) before it went it. I'll update this one, thanks.